src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -16,16 +16,20 @@
 
     <div class="container">
       <div class="columns">
-
+        <img src= "lowcarbdiet.jpg", width="600", height="300">
         <div class="column col-8-col-mx-auto">
-
             <div class="hero hero-sm bg-dark">
                 <div class="hero-body">
                     <h1>FoodO2</h1>
                     <p>By The Carbon Mob</p>
                 </div>
             </div>
 
+            <p>Welcome! This is our FoodCO2 APP, where you can calculate the carbon footprint of your diet
+      and keep track of how much greenhouse gas emissions you contribute through your consumption.
+      Overall, we hope that this APP will raise awarenss about the environmental impact of different food choices and encourage
```

Comment:
awarenss --> awareness

Reply:
Weird this isn't even from my commits, It's from a rebase of Ivy's changes